### DISCUSSION TITLE
Rutas

### DISCUSSION THREAD

**Discussion by LuisFerVR** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#474] [Q&A]
![image](https://user-images.githubusercontent.com/98798741/159191083-3019e615-f8f5-40ca-bfa0-773004178e04.png)
![image](https://user-images.githubusercontent.com/98798741/159191086-2d7492bb-abe0-4409-af53-7647f0d9b9cf.png)
Disculpen, saben cómo solucionar ese error que me parece? aparece al querer agregar varias rutas

**Comment by doguedogue** [comment]
Desactivar eslint podría ser una opción si estás usando las rutas generadas por el proyecto.

**Comment by AngelCruzO** [comment]
Yo lo solucione, metiendo todo en un div sin clase

**Comment by Ingrzs** [comment]
@doguedogue @AngelCruzO de casualidad sabran porque a mi no aparece el código coloreado? Es como si las extensiones que tengo instaladas no las detectará, puedo hacer todo pero todo el código tiene letra color blanca y es difícil distinguir donde termina esto y lo otro

**Reply by doguedogue** [reply]
Hay muchas extensiones que te ayudan en visual code.

Creo que la que te ayuda a eso es **Prettier**, hay otra para que tú definas el color **Bracket Pair colorizer**, etc.

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

https://marketplace.visualstudio.com/items?itemName=CoenraadS.bracket-pair-colorizer-2

**Reply by Ingrzs** [reply]
@doguedogue de echo ya tengo instaladas esas extensiones y si funciona  en css ,js, html  pero ya creando el archivo vuex y todo eso pareciera que no las tengo instaladas

**Reply by doguedogue** [reply]
Ah ok, la vieja confiable, activarlos y desactivarlos.

Las extensiones vue si me las pone de colores.

Aquí otras extensiones
https://learnvue.co/2021/01/the-7-best-vs-code-extensions-for-vue-developers/#bookmarks

**Reply by AngelCruzO** [reply]
Yo tengo instaladas Vetur(Pine Wu) y vue (jcbuisson), no se si te sirvan para que resalte la sintaxis
